en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -90,8 +88,6 @@ Here is how it works in detail:
    Even if `$CacheKey` is omitted, `SilverStripe\View\SSViewer::$global_key` and `Block hash` values are still
    getting used to generate cache key for the caching backend storage.
 
-[note]
-
```

Comment:
Took this below content out of the callout block - it's got a heading, so it should just be its own section.